.circleci/config.yml

```diff
@@ -430,6 +432,7 @@ jobs:
       - run:
           name: Install Device Farm Dependencies
           command: make install-devicefarm-dependencies
+      - install-dependencies
```

Comment:
Is this necessary?

Reply:
Yes, because this causes xcodegen to be installed, and on line 450, `make test-with-device-farm` is invoked, which runs xcodegen unconditionally since it's shared between unit tests and examples tests.